app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] great job so far. Please kindly ensure Uses CanCanCan for Post authorization in your Post Controller
```ruby
 class PostsController < ApplicationController
    load_and_authorize_resource
```

Reply:
@tufoinnkuo10 noted